src/test/java/tests/FilterTest.java

```diff
@@ -0,0 +1,39 @@
+package tests;
+
+import indices.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.ApplePage;
+import pages.DominoPage;
+import pages.MainPage;
+import pages.SmartphonePage;
+import testData.Catalogue;
+
+
+public class FilterTest extends BaseTest {
+
+    @Test(description = "Positive test to choose Apple manufacturer in the filter")
```

Comment:
It's not necessary to mention Positive or Negative test in the description, you can just add test description there.
Also it's good practice ass TC id to test description, it help with searching for the tests

Reply:
done